File: SDKs/Aspose.Pdf_Cloud_SDK_for_CSharp/src/Com/Aspose/PDF/Model/PageWordCount.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class PageWordCount {
    public int? PageNumber { get; set; }

    public int? Count { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class PageWordCount {\n");
      sb.Append("  PageNumber: ").Append(PageNumber).Append("\n");
      sb.Append("  Count: ").Append(Count).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Android/Aspose-PDF-Cloud-SDK-for-Android/src/main/java/com/aspose/pdf/model/PageWordCount.java
package com.aspose.pdf.model;

public class PageWordCount {
  private Integer PageNumber = null;
  private Integer Count = null;
  /**
	 * getPageNumber
	 * Gets Integer
	 * @return PageNumber
	 */
  public Integer getPageNumber() {
    return PageNumber;
  }

	/**
	 * setPageNumber
	 * Sets Integer
	 * @param PageNumber Integer
	 */
  public void setPageNumber(Integer PageNumber) {
    this.PageNumber = PageNumber;
  }

  /**
	 * getCount
	 * Gets Integer
	 * @return Count
	 */
  public Integer getCount() {
    return Count;
  }

	/**
	 * setCount
	 * Sets Integer
	 * @param Count Integer
	 */
  public void setCount(Integer Count) {
    this.Count = Count;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class PageWordCount {\n");
    sb.append("  PageNumber: ").append(PageNumber).append("\n");
    sb.append("  Count: ").append(Count).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

